olive/workflows/run/run.py

```diff
@@ -183,3 +192,56 @@ def run(config: Union[str, Path, dict], setup: bool = False, data_root: str = No
             config.engine.evaluate_input_model,
         )
         return best_execution
+
+
+def check_local_ort_installation(package_name: str):
+    local_ort_packages = get_local_ort_packages()
+
+    if not local_ort_packages:
+        # this case should not happen right now, for future proofing
+        # onnxruntime is a dependency of olive so it must already be present
+        # olive import would not have succeeded otherwise
+        logger.info(f"Installing {package_name}...")
+        subprocess.check_call(["python", "-m", "pip", "install", package_name])
+        logger.info(f"Successfully installed {package_name}.")
+        return
+
+    if "-" in package_name:
+        night_package_name = f"ort-nightly-{package_name.split('-')[-1]}"
+    else:
+        night_package_name = "ort-nightly"
+
+    if len(local_ort_packages) == 1 and local_ort_packages[0] in [package_name, night_package_name]:
+        # only if one ort package is installed and it is the one we want
+        # can be the stable or nightly version
+        # TODO: will probably be fine if we want cpu package but some other ort package is installed
+        # but we can add a check for that if needed in the future
+        logger.info(f"{local_ort_packages[0]} is already installed.")
+        return
+
+    # uninstall all ort packages
+    logger.info("There are one or more onnxruntime packages installed in your environment.")
+    uninstall_command = ["python", "-m", "pip", "uninstall", "-y"] + local_ort_packages
+    uninstall_command = " ".join(uninstall_command)
+    logger.info(f"Please run '{uninstall_command}' to uninstall all existing onnxruntime packages.")
+
+    # install the package
+    logger.info(f"Then install {package_name} by running 'python -m pip install {package_name}'.")
```

Comment:
forget to install?

Reply:
We only log the command to the user. 
Since they have to first uninstall all other ort packages and then install the correct one

Reply:
I had a discussion with Xiaoyu above and we thought it's better for olive to not uninstall packages and just tell the user

Reply:
Updated after offline discussion